### PR TITLE
feature: refactor and add banner to storybook

### PR DESCRIPTION
**Pull request by knav33n**
## Problem

[1243](https://github.com/tailwarden/komiser/issues/1243 "Add Banners to Storybook")

## Changes Made
- Refactored Banner into a generic component and added it to Storybook.
- Created GithubBanner by extending the generic Banner component.

## Screenshots
banner-storybook

## Checklist

- [x] Code follows the <a href="https://github.com/tailwarden/komiser/blob/master/CONTRIBUTING.md">contributing</a> guidelines
- [x] Changes have been thoroughly tested
- [ ] <a href="https://github.com/tailwarden/docs.komiser.io">Documentation</a> has been updated, if necessary
- [ ] Any dependencies have been added to the project, if necessary

## Reviewers

@Traxmaxx @Azanul @AllieMendes @mlabouardy

### CODE REVIEW

File: dashboard/components/layout/hooks/useGithubStarBanner.tsx [hunk 1]
```diff
@@ -35,7 +35,7 @@ function useGithubStarBanner() {
   useEffect(() => {
     const shouldDisplayBanner = checkLocalStorageForBannerStatus();
 
-    if (shouldDisplayBanner !== 'false' && router.query.view) {
+    if (true) {
```
Comment (AvineshTripathi) [inline dashboard/components/layout/hooks/useGithubStarBanner.tsx:38]: why are we making this true?

**Review by AvineshTripathi — COMMENTED** [review]
@Azanul to do detail review

File: dashboard/components/layout/hooks/useGithubStarBanner.tsx (hunk 1 above)
Comment (knav33n) [reply dashboard/components/layout/hooks/useGithubStarBanner.tsx]: My apologies – I used this for some last-minute testing.

**Review by AvineshTripathi — COMMENTED** [review]
LGTM 

@Azanul  please review the frontend changes! I saw the working and overview looks good to me

**Review by Azanul — APPROVED** [review]

**Comment by Azanul** [comment]
@AvineshTripathi Please approve the PR if everything looks

**Review by AvineshTripathi — APPROVED** [review]

**Pull request merged**